features/steps/steps.py

```diff
@@ -75,9 +75,20 @@ def __str__(self):
 class instance_structure_error:
     related: ifcopenshell.entity_instance
     relating: ifcopenshell.entity_instance
+    relationship_type: str
+    optional_values: field(default_factory=dict)
 
     def __str__(self):
-        return f"The instance {fmt(self.related)} is assigned to {fmt(self.relating)}"
+        self.pos_neg = 'is not' if do_try(lambda: self.optional_values['condition'], '') == 'must' else 'is'
+        self.directness = do_try(lambda: self.optional_values['directness'],'')
+        
+        if len(self.relating):
+            if len(self.relating) > 1:
```

Comment:
It was because the wording is slightly different when it is a single case as opposed to more entities in the error log. The message in this error log is changed now, it's less confusing and will also take into consideration (future) cases in which 'pos_neg' and 'directness' (now in ALB001) are linked to more entities.